README.md

```diff
@@ -2,59 +2,49 @@
 Andromeda is a website that allows a user to perform dimensional reduction on an uploaded CSV file.
 
 ## Requirements
-Deployment requires [docker-compose](https://www.docker.com/).
+Deployment requires [Docker](https://www.docker.com/).
 
 Development requires [python3](https://www.python.org/) and [nodejs](https://nodejs.org/).
 
 ## Deployment
-To deploy the website on a machine running docker run the following commands:
-```bash
-git clone https://github.com/Imageomics/Andromeda.git
-cd Andromeda/
-cp example.env .env
-docker-compose up -d
-```
-The `-d` flag runs the website in the background. For troubleshooting remove this flag to more easily monitor the deployment.
+Our primary deployment environment is [Hugging Face Spaces](https://huggingface.co/spaces/imageomics/Andromeda).
 
-Once the website finishes launching the website will be available on port 80: [http://localhost](http://localhost).
-Part of the deployment builds the docker containers used by the website, so it may take a few minutes to finish deployment.
-To stop the app run `docker-compose down`.
+To build a Docker image from the source code and deploy the container:
```

Comment:
The following steps do not deploy the container. Could we move these instructions to a developing with docker section? I would like this section to explain how to deploy a new version of Andromeda. My thought would be something along the lines of 
1) Create a new GitHub Release
2) Wait for the container to build
3) Update the version in 
https://huggingface.co/spaces/imageomics/Andromeda/blob/main/Dockerfile

Reply:
I like the idea of moving this into the wiki and adding a link.